src/tests/auth/authz/genericSetup.js

```diff
@@ -0,0 +1,30 @@
+import {getAuthToken} from "../../../common/utils.js";
+import {logErrorResult} from "../../../common/dynamicScenarios/utils.js";
+import {CONFIG} from "../../../common/envVars.js";
+import {getOrganizationOperators} from "../../../api/auth/authz/auth.js";
+
+// Utility function to set up the test environment
+export function setupTestEnvironment(testName) {
```

Comment:
let's use more significant name
```suggestion
export function setupOperatorBasedTest(testName) {
```

Reply:
Modified